modules/navier_stokes/test/tests/finite_volume/cns/mms/1d/no-diffusion-hllc-mms.i

```diff
@@ -0,0 +1,192 @@
+[GlobalParams]
+  fp = fp
+[]
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 1
+    xmin = .1
+    xmax = 1
+    nx = 2
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
+
+[Variables]
+  [rho]
+    type = MooseVariableFVReal
+  []
+  [rho_u]
+    type = MooseVariableFVReal
+  []
+  [rho_et]
+    type = MooseVariableFVReal
+  []
+[]
+
+[ICs]
+  [rho]
+    type = FunctionIC
+    variable = rho
+    function = 'cos(1.1*x)'
+  []
+  [rho_u]
+    type = FunctionIC
+    variable = rho_u
+    function = '2*sin(1.1*x)'
+  []
+  [rho_et]
+    type = FunctionIC
+    variable = rho_et
+    function = '3*cos(1.1*x)'
+  []
+[]
+
+[FVKernels]
+  [mass_advection]
+    type = CNSFVMassHLLC
+    variable = rho
+  []
+  [mass_fn]
+    type = FVBodyForce
+    variable = rho
+    function = 'forcing_rho'
+  []
+  [momentum_advection]
+    type = CNSFVMomentumHLLC
+    variable = rho_u
+    momentum_component = x
+  []
+  [momentum_fn]
+    type = FVBodyForce
+    variable = rho_u
+    function = 'forcing_rho_u'
+  []
+  [fluid_energy_advection]
+    type = CNSFVFluidEnergyHLLC
+    variable = rho_et
+  []
+  [energy_fn]
+    type = FVBodyForce
+    variable = rho_et
+    function = 'forcing_rho_et'
+  []
+[]
+
+[FVBCs]
+  [rho]
+    type = FVFunctionDirichletBC
+    boundary = 'left right'
+    variable = rho
+    function = 'exact_rho'
+  []
+  [rho_u]
+    type = FVFunctionDirichletBC
+    boundary = 'left right'
+    variable = rho_u
+    function = 'exact_rho_u'
+  []
+  [rho_et]
+    type = FVFunctionDirichletBC
+    boundary = 'left right'
+    variable = rho_et
+    function = 'exact_rho_et'
+  []
+[]
+
+[Materials]
+  [var_mat]
+    type = ConservedVarValuesMaterial
+    rho = rho
+    rhou = rho_u
+    rho_et = rho_et
+  []
+[]
+
+[Modules]
+  [FluidProperties]
+    [fp]
+      type = TestConservedVarFluidProperties
+    []
+  []
+[]
+
+[Functions]
+[exact_rho]
+  type = ParsedFunction
+  value = 'cos(x)'
+[]
+[forcing_rho]
+  type = ParsedFunction
+  value = '2*cos(x)'
+[]
+[exact_rho_u]
+  type = ParsedFunction
+  value = '2*sin(x)'
+[]
+[forcing_rho_u]
+  type = ParsedFunction
+  value = '4*sin(x)^3/cos(x)^2 + 7*sin(x)'
+[]
+[exact_rho_et]
+  type = ParsedFunction
+  value = '3*cos(x)'
+[]
+[forcing_rho_et]
+  type = ParsedFunction
+  value = '8*cos(x)'
+[]
+[]
+
+[Executioner]
+  type = Steady
+  solve_type = 'NEWTON'
+  petsc_options_iname = '-pc_type'
+  petsc_options_value = 'lu'
+  # petsc_options = '-pc_svd_monitor'
+  # petsc_options_iname = '-pc_type'
+  # petsc_options_value = 'svd'
```

Comment:
```suggestion
```
anything on the dominance ratio you learned from these?

Reply:
I think this was just for debugging something wrong in the initial implementation where things were singular when they shouldn't have been